catalog_classes/genre.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './item'
+
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item.genre = self
+  end
```

Comment:
Your implementation is remarkable, however, it allows for an existing item to be added multiple times. kindly I think you should write a condition to ensure this isn't the case.
 such as  item unless @items.include?(item)